packages/api/src/external/aws/appConfig.ts

```diff
@@ -18,24 +18,38 @@ export async function getCxsWithEnhancedCoverageFeatureFlagValue(): Promise<stri
       Config.getAppConfigAppId(),
       Config.getAppConfigConfigId(),
       Config.getEnvType(),
-      Config.getCxsWithEnhancedCoverageFeatureFlagName()
+      featureFlagName
     );
     if (featureFlag?.enabled && featureFlag?.cxIds) return featureFlag.cxIds;
   } catch (error) {
     console.log(
-      `Failed to get cxsWithEnhancedCoverage Feature Flag Value with error: ${errorToString(error)}`
+      `Failed to get ${featureFlagName} Feature Flag Value with error: ${errorToString(error)}`
     );
     capture.error(error, {
```

Comment:
Since we're touching this, can we move this to the pattern to have errors notified as the business logic message instead of the error message, and also keep the error message static (no parameters when building the string):

```ts
        const msg = `Failed to get Feature Flag Value`;
        const extra = { featureFlagName };
        log(`${msg} - ${JSON.stringify(extra)} - ${errorToString(error)}`);
        capture.error(msg, { extra: { ...extra, error } });
```